src/mscorlib/shared/System/IO/FileStream.Unix.cs

```diff
@@ -796,6 +796,22 @@ private Task WriteAsyncInternal(byte[] buffer, int offset, int count, Cancellati
             }
         }
 
+        /// <summary>Prevents other processes from reading from or writing to the FileStream.</summary>
+        /// <param name="position">The beginning of the range to lock.</param>
+        /// <param name="length">The range to be locked.</param>
+        private void LockInternal(long position, long length)
```

Comment:
Do we need this back? I though it got stripped because of it was not used anywhere.